site/content/en/docs/Reference/gameserverallocation.md

```diff
@@ -78,13 +78,7 @@ spec:
     matchLabels:
       game: my-game
     matchExpressions:
-      - {key: tier, operator: In, values: [cache]}
-    # [Stage:Beta]
-    # [FeatureFlag:StateAllocationFilter]
-    # Specifies which State is the filter to be used when attempting to retrieve a GameServer
-    # via Allocation. Defaults to "Ready". The only other option is "Allocated", which can be used in conjunction with
-    # label/annotation/player selectors to retrieve an already Allocated GameServer.    
-    gameServerState: Ready
```

Comment:
This should not be removed.

Reply:
Should also remove line 39 and 40

Reply:
Restored the code and removed the lines 39 and 40.